metriq_gym/helpers/graph_helpers.py

```diff
@@ -76,7 +76,17 @@ def device_graph_coloring(topology_graph: rx.PyGraph) -> GraphColoring:
     if rx.is_bipartite(topology_graph):
         edge_color_map = rx.graph_bipartite_edge_color(topology_graph)
     else:
-        edge_color_map = rx.graph_greedy_edge_color(topology_graph)
+        # Check if graph is complete
+        num_edges = len(topology_graph.edge_list())
+        expected_edges_if_complete = num_nodes * (num_nodes - 1) // 2
+        is_complete = num_edges == expected_edges_if_complete
+
+        if is_complete:
```

Comment:
Can we have a unit test for these two cases?
It should be trivial to write. We can explicitly create a complete graph, pass it to this function and assert the right algorithm is called.